vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```
@@ -42,7 +42,7 @@
       state: shutdown
     register: shutdown_vms
     retries: 50
```

As we have decreased the delay I think we need to increase the retries to maintain the same time needed to wait ? 
```suggestion
    retries: 166
```

---

otherwise that many retries may not be enough too :)

---

Thanks, fixed!